belvedere/src/main/res/values/strings.xml

```diff
@@ -1,19 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <resources>
-	<string name="belvedere_dialog_camera">Camera</string>
-	<string name="belvedere_dialog_gallery">Gallery</string>
-	<string name="belvedere_fam_desc_collapse_fam">Collapse Floating Action Menu</string>
-	<string name="belvedere_fam_desc_expand_fam">Expand Floating Action Menu</string>
-	<string name="belvedere_fam_desc_open_gallery">Import file from gallery</string>
-	<string name="belvedere_fam_desc_open_google_photos">Import image from Google Photos</string>
-	<string name="belvedere_image_stream_file_too_large">File too large</string>
-	<string name="belvedere_image_stream_title">Photo library</string>
-	<string name="belvedere_image_stream_unknown_app">Unknown</string>
-	<string name="belvedere_permissions_denied">Permissions denied</string>
-	<string name="belvedere_stream_item_camera_tile_desc">Import image from camera</string>
-	<string name="belvedere_stream_item_select_file_desc">Select file %s</string>
-	<string name="belvedere_stream_item_select_image_desc">Select image %s</string>
-	<string name="belvedere_stream_item_unselect_file_desc">Unselect file %s</string>
-	<string name="belvedere_stream_item_unselect_image_desc">Unselect image %s</string>
-	<string name="belvedere_toolbar_desc_collapse">Close image picker</string>
+    <string name="belvedere_dialog_camera">Camera</string>
+    <string name="belvedere_dialog_gallery">Gallery</string>
+    <string name="belvedere_fam_desc_collapse_fam">Collapse Floating Action Menu</string>
+    <string name="belvedere_fam_desc_expand_fam">Expand Floating Action Menu</string>
+    <string name="belvedere_fam_desc_open_gallery">Import file from gallery</string>
+    <string name="belvedere_fam_desc_open_google_photos">Import image from Google Photos</string>
+    <string name="belvedere_image_stream_file_too_large">File too large</string>
+    <string name="belvedere_image_stream_title">Photo library</string>
+    <string name="belvedere_image_stream_unknown_app">Unknown</string>
+    <string name="belvedere_permissions_denied">To send attachments, you must grant permissions in your settings</string>
+    <string name="belvedere_navigate_to_settings" translatable="false">Settings</string>
```

Comment:
Quick question: Does the build fail if we don't mark "Settings" as non-translatable.?

Reply:
Just for now, we should translate it for all the supported languages we have in the library. I don't know how we can do that, yet that's why I labeled it as non-translatable for now until we translate it in another PR.